core/lib/resources/presence/index.js

```diff
@@ -175,7 +179,7 @@ Presence.prototype.unsubscribe = function (socket, message) {
 Presence.prototype.sync = function (socket, message) {
   var self = this
   this.fullRead(function (online) {
-    if (message.options && message.options.version == 2) {
+    if (message.options && message.options.version === 2) {
```

Comment:
same problems as before. This could come in as a string and it should still be fine.